UnitsNet.Tests/CustomCode/DurationTests.cs

```diff
@@ -33,20 +33,6 @@ public class DurationTests : DurationTestsBase
 
         protected override double JulianYearsInOneSecond => 3.16880878140289e-08;
 
-        [Fact]
-        public static void ToTimeSpanShouldThrowExceptionOnValuesLargerThanTimeSpanMax()
-        {
-            Duration duration = Duration.FromSeconds(TimeSpan.MaxValue.TotalSeconds + 1);
-            Assert.Throws<ArgumentOutOfRangeException>(() => duration.ToTimeSpan());
-        }
-
-        [Fact]
-        public static void ToTimeSpanShouldThrowExceptionOnValuesSmallerThanTimeSpanMin()
-        {
-            Duration duration = Duration.FromSeconds(TimeSpan.MinValue.TotalSeconds - 1);
-            Assert.Throws<ArgumentOutOfRangeException>(() => duration.ToTimeSpan());
-        }
-
```

Comment:
We can keep these tests, no? I would still expect an exception in these cases.

Reply:
My bad, I removed some checks that seemed redundant but it appears TimeSpan simply rolls over to negative values if exceeding the max value.

I don't like that behavior, so I'd rather add these checks back.

Reply:
Added back